scripts/variantstore/beta_docs/gvs-cleanup.md

```diff
@@ -0,0 +1,14 @@
+# Genomic Variant Store Cleanup
+
+## Use Case 1: One-Off Callset
+### Assumptions
+- the samples in the callset are going to be joint called once (no sub-cohorts, no re-using the model)
+- the outputs (VCFs, indexes, interval lists, manifest, sample name list) have all been copied to an independent location (not in the workspace bucket)
+- no need to maintain provenance
+- the workspace where the pipeline was run is only being used for this GVS callset
+
+### Actions
```

Comment:
Sounds good - Maybe??? need a more explicit: "CAUTION - This will cause permanent deletion of data - check with hOPs if you have any questions about this"

Reply:
I agree a note of caution should be added.  We should confirm that it's hOPs they should ask, though.  @kbergin thoughts?

Reply:
I think it's sufficient to just say "CAUTION- This will result in permanent deletion of data."